kata-deploy/k8s-1.13/kata-qemu-runtimeClass.yaml

```diff
@@ -2,11 +2,13 @@
 kind: RuntimeClass
 apiVersion: node.k8s.io/v1alpha1
 metadata:
-    name: kata
-handler: kata
+    name: kata-qemu
+spec:
+    runtimeHandler: kata-qemu
```

Comment:
I'm presuming the order swap of `kata` and `kata-qemu` here is benign?

Reply:
yes